Designs/change_password/asset/style.scss

```
@@ -0,0 +1,29 @@
+.container{
+    
+    .card{
+        margin:0 auto;
+        max-width: 400px;
+        background-color: #b7b4b4;
```

Change background color

```suggestion
        background-color: #f7f7f7;
```